jcli/src/jcli_lib/rest/config.rs

```diff
@@ -7,12 +7,12 @@ use std::path::PathBuf;
 use structopt::StructOpt;
 use thiserror::Error;
 
-#[derive(StructOpt)]
+#[derive(StructOpt, Clone)]
 pub struct RestArgs {
     /// node API address. Must always have `http://` or `https://` prefix.
     /// E.g. `-h http://127.0.0.1`, `--host https://node.com:8443/cardano/api`
     #[structopt(short, long, env = "JORMUNGANDR_RESTAPI_URL")]
-    host: Url,
+    pub host: Url,
```

Comment:
I needed to access that field that otherwise is just for internal use. If embedded in some other command (as happens here) it is handy to access it. Do you think a method would be better?

Reply:
> If embedded in some other command (as happens here) it is handy to access it.

Fine if that's the case :+1: